### PR TITLE
[ios , macos] Exclude CJK resources from offline download by default

### PR DESCRIPTION
**Pull request by m-stephen**
Closes: #14176
After this PR: https://github.com/mapbox/mapbox-gl-native/pull/14862 merged, we exclude CJK resources from offline download by default.

### CODE REVIEW

**Comment by m-stephen** [comment]
cc: @chloekraw @julianrex

**Comment by chloekraw** [comment]
Thanks for this PR, @m-stephen. Is this blocked by https://github.com/mapbox/mapbox-gl-native/issues/14271?

**Comment by m-stephen** [comment]
> Thanks for this PR, @m-stephen. Is this blocked by #14271?

@chloekraw I don't think so, just on iOS side. I wrote a offline pack demo today and make a clean(no cache) app for testing:

- Bounds: [LatLng(31.306819,120.578003), LatLng(31.32,120.59)]
- MinZoom: 1
- MaxZoom: 16
- Style: MAPBOX_STREETS
- Include ideographs: PingFang TC

And after download these offline resources, I switch to airplane mode and I think it rendered well:
image

cc: @zugaldia

**Comment by m-stephen** [comment]
Btw, this PR should be approved after #14862 merged because we have changed `MGLIdeographicFontFamilyName` to a multiple value, especially when it is changed to a `Boolean` value, which means we turn off the local rendering.

**Comment by friedbunny** [comment]
Are there unit tests that can be written for this?

Check out  [`platform/darwin/test/MGLOfflineStorageTests.mm`](https://github.com/mapbox/mapbox-gl-native/blob/24df7fe5f518dfe3dfbdb138bb12a785251d2395/platform/darwin/test/MGLOfflineStorageTests.mm) for one existing implementation of an offline pack used as a test fixture.

**Comment by m-stephen** [comment]
> Are there unit tests that can be written for this?
> 
> Check out [`platform/darwin/test/MGLOfflineStorageTests.mm`](https://github.com/mapbox/mapbox-gl-native/blob/24df7fe5f518dfe3dfbdb138bb12a785251d2395/platform/darwin/test/MGLOfflineStorageTests.mm) for one existing implementation of an offline pack used as a test fixture.

Sure, will do.

**Comment by julianrex** [comment]
@m-stephen how are you doing with tests for this?

**Comment by m-stephen** [comment]
@julianrex Sorry for forgetting upload test codes, already uploaded now. Please review it, thanks!

File: platform/ios/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## master
+
+* Exclude CJK resources from offline download by default. ([#14176](https://github.com/mapbox/mapbox-gl-native/pull/14176))
```
Comment (friedbunny) [inline platform/ios/CHANGELOG.md]:
```suggestion
* Ideographic glyphs from Chinese, Japanese, and Korean are no longer downloaded by default as part of offline packs; they are instead rendered on-device, saving bandwidth and storage while improving performance. ([#14176](https://github.com/mapbox/mapbox-gl-native/pull/14176))
```

File: platform/macos/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 * Added an `MGLMapView.prefetchesTiles` property to configure lower-resolution tile prefetching behavior. ([#14816](https://github.com/mapbox/mapbox-gl-native/pull/14816))
 * Fixed queryRenderedFeatues bug caused by incorrect sort feature index calculation. ([#14884](https://github.com/mapbox/mapbox-gl-native/pull/14884))
+* Exclude CJK resources from offline download by default. ([#14176](https://github.com/mapbox/mapbox-gl-native/pull/14176))
```
Comment (friedbunny) [inline platform/macos/CHANGELOG.md]:
```suggestion
* Ideographic glyphs from Chinese, Japanese, and Korean are no longer downloaded by default as part of offline packs; they are instead rendered on-device, saving bandwidth and storage while improving performance. ([#14176](https://github.com/mapbox/mapbox-gl-native/pull/14176))
```

**Comment by LukasPaczos** [comment]
In https://github.com/mapbox/mapbox-gl-native/pull/14952#issuecomment-503494054 you mentioned:
> Include ideographs: PingFang TC

@m-stephen do you mean that your local font family for glyphs generation is set to `PingFang TC`, right? The downloaded region still doesn't include any ideographs?

**Comment by m-stephen** [comment]
> @m-stephen do you mean that your local font family for glyphs generation is set to PingFang TC, right? The downloaded region still doesn't include any ideographs?

Yes. For clarifying this case, the `PingFang TC` is the local font family name, and setting the offline region's `Include ideographs` to `NO`.  I didn't get any rendering issue on my phone.

**Comment by chloekraw** [comment]
@m-stephen - we'd love to get this PR in before the `release-p` beta branch cut tomorrow. Could you resolve the conflicts and nits in this PR so we can get it merged? Thanks!

cc @chriswu42 @mapbox/maps-ios

**Comment by m-stephen** [comment]
> @m-stephen - we'd love to get this PR in before the release-p beta branch cut tomorrow. Could you resolve the conflicts and nits in this PR so we can get it merged? Thanks!

Sorry for forgetting to merge this, already merged! @chloekraw

**Comment by chloekraw** [comment]
No problem - thanks!

**Pull request merged**